src/java/org/apache/cassandra/tools/nodetool/Snapshot.java

```diff
@@ -73,6 +74,10 @@ public void execute(NodeProbe probe)
                 options.put("ttl", d.toString());
             }
 
+            if (!snapshotName.isEmpty() && snapshotName.contains(File.pathSeparator()))
```

Comment:
is path separator the only character illegal to be included in a snapshot name?

Reply:
What about null character "\0" ?

Reply:
@skoppu22 I would like to have that discussion in the ticket. Ticket is the must anyway in order to merge it. No way around it.

Reply:
@smiklosovic I am not the author of the PR